adapters/adapters-commons/src/main/java/uk/gov/justice/raml/common/mapper/ActionMapping.java

```diff
@@ -77,6 +91,21 @@
         return emptyList();
     }
 
+    private static String extractMappingFrom(final String mappingString) {
+        return removeUpToIncludingMappingBoundary(removeAfterIncludingMappingBoundary(mappingString));
+    }
+
+    private static String removeUpToIncludingMappingBoundary(final String source) {
```

Comment:
Would trimPreMappingDescription() and trimPostMappingDescription() be clearer names? - or is there a name we can use which is a little clearer?